Source/cielim/Private/RenderingFunctionsLibrary.cpp

```diff
@@ -110,7 +110,7 @@ void URenderingFunctionsLibrary::ApplyReadNoise(TArray<uint8>& ImageData, float
 	cv::imwrite(ResultFilepath_String, ResultImage);
 
 	// Encode corrupted image as png
-    std::vector<uint8_t> EncodedData;
+    std::vector<uint8> EncodedData;
```

Comment:
:large_orange_diamond: Is there a reason for preferring the `uint8` type over the `uint8_t` type? I've normally chosen the later because it guarantees type widths across different platforms. I've likely forgotten this in places and done the same.

Reply:
I believe that uint8 is a type specifically for use in Unreal, so I am using that. They should be effectively equivalent.